tembo-stacks/src/stacks/types.rs

```diff
@@ -175,6 +214,20 @@ fn default_storage() -> String {
     "10Gi".to_owned()
 }
 
+pub fn merge_options<T>(opt1: Option<Vec<T>>, opt2: Option<Vec<T>>) -> Option<Vec<T>>
```

Comment:
Unless I missed something, this isn't called in the changes in this review, right? What's it for?